docs/proposals/EntGAN.md

```diff
@@ -0,0 +1,43 @@
+# Design and Implement of GAN based on Sedna Lifelong Learning
+## Motivation
+
+At present, robot collision detection requires a large number of images of multiple types as training data. However, to my best knowledge, even the most prevalent and largest public dataset([the cityscapes dataset](https://www.cityscapes-dataset.com/) and [the synthia dataset](https://synthia-dataset.net/)) cannot satisfy it.    
+Generate Adversarial Networks(GAN) is the start-of-art generative model and GAN can generate fake data according to the distribution of the real data. Naturally, GAN is used to learn the distribution of the current data, thereby generating more data to train the robot collision detection. Nevertheless, GAN is hard to train and converge. At the same time, GAN that can generate multiple types of data is needed, because of the characteristic of robot collision detection.    
+Therefore, we design a GAN framework based on sedna lifelong learning due to the ingenious design of Sedna. We call it EntGAN.
+### Goals
```

Comment:
The motivation description is abstract. Please clarify why edge-cloud synergy is required.

Reply:
+1